freqtrade/rpc/api_server.py

```diff
@@ -358,6 +359,19 @@ def _balance(self):
                                     self._config.get('fiat_display_currency', ''))
         return self.rest_dump(results)
 
+    @require_login
+    @rpc_catch_errors
+    def _trades(self):
+        """
+        Handler for /trades.
+
+        Returns the X last trades in json format
+        """
+        last_trades_number = request.args.get('last_trades_number', 0)
+        last_trades_number = int(last_trades_number)
```

Comment:
You should be able to combine these 2 lines - it's not really necessary to have these in 2 lines i think.

```suggestion
        last_trades_number = int(request.args.get('last_trades_number', 0))
```

Reply:
Yeah it's useless